src/App.tsx

```diff
@@ -1,8 +1,38 @@
 import React from 'react';
 import { Login } from './views/Login';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Dashboard } from 'views/Dashboard';
+import { Leads } from 'views/Leads';
+import { RequireAuth, useIsAuthenticated } from 'react-auth-kit';
 
 function App() {
-   return <Login />;
+   const isAuthenticated = useIsAuthenticated();
+
+   return (
+      <Routes>
+         <Route path="/" element={<Navigate to="/login" />} />
+         <Route
+            path="/login"
+            element={!isAuthenticated() ? <Login /> : <Navigate to="/dashboard" />}
+         />
+         <Route
```

Comment:
**nitpick:** Can't you wrap these routes into `  <RequireAuth>`? It will be simpler. Eg.

```tsx
  <RequireAuth loginPath="/login">
  <Route
            path="/dashboard"
            element={<Dashboard />}
         />
  <Route
            path="/leads"
            element={<Leads />}
         />
<RequireAuth />
```

Reply:
@Sumick  Unfortunately I can't wrap these routes like you proposed becouse `<RequireAuth>` accepts only one element as children